src/random.ts

```diff
@@ -266,19 +257,7 @@ function asyncRefillRandomIntCache() {
   });
 }
 
-// Really just the Web Crypto API alternative
-// to require('crypto').randomFillSync() with an
-// additional limitation that the input buffer is
-// not allowed to exceed 65536 bytes, and can only
-// be an integer-type TypedArray.
-export type RandomTypedArrays =
-  | Int8Array
-  | Int16Array
-  | Int32Array
-  | Uint8Array
-  | Uint16Array
-  | Uint32Array;
-export function getRandomValues(data: RandomTypedArrays) {
+export function getRandomValues(data: TypedArray) {
```

Comment:
neither Float32Array nor Float64Array should be allowable here, that's why `RandomTypedArrays` existed. this should be throwing a type error and should not have compiled with `tsc` since it conflicts with Node's `getRandomValues`

Reply:
Oh, that's a good point, @boorad what do you think? I can create a different PR were we export `RandomTypedArrays` again and use it for `getRandomValues`

Reply:
yes plz 🙏

Reply:
To be fair I don't think the change here is super harmful but I'm more curious how tsc was able to compile given the type error. I think the bigger concern is why there wasn't a type error here

Reply:
<img width="842" alt="Screenshot 2024-05-30 alle 18 36 21" src="https://github.com/margelo/react-native-quick-crypto/assets/17269969/c2bb3111-46c8-45b5-8b63-90ebd203f2a7">

I think because `randomFillSync` just extend it, and it also use `any`:
<img width="621" alt="Screenshot 2024-05-30 alle 18 37 54" src="https://github.com/margelo/react-native-quick-crypto/assets/17269969/e654987c-6494-44e4-83f6-b28caa246ece">

Reply:
I'll create anyway the PR, because even if typescript doesn't complain about that it's a wrong type definition

Reply:
Done here: https://github.com/margelo/react-native-quick-crypto/pull/330